File: projects/web-api/Thinktecture.Boardist.WebApi/Database/Models/Publisher.cs
using System;

namespace Thinktecture.Boardist.WebApi.Database.Models
{
  public class Publisher : Syncable, IBoardGameGeekItem
  {
    public string Name { get; set; }
    public int Priority { get; set; }

    public int? BoardGameGeekId { get; set; }
  }
}

File: projects/java-web-api/src/main/java/com/thinktecture/boardist/webApi/models/Publisher.java
package com.thinktecture.boardist.webApi.models;

import java.util.UUID;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;
import org.hibernate.annotations.GenericGenerator;

@Entity()
@Table(name = "Publishers", schema = "dbo", catalog = "boardist")
public class Publisher extends Item {
    
    @Id
    @GenericGenerator(name = "uuid", strategy = "uuid2")
    @GeneratedValue(generator = "uuid")
    @Column(columnDefinition = "UNIQUEIDENTIFIER")
    public UUID id;
    
    public String name;
    public int boardGameGeekId;
    public byte[] rowVersion;
    public boolean isDeleted;

    public int priority;
}
